.github/workflows/main.yml

```diff
@@ -0,0 +1,164 @@
+      - name: Installer file name
+        id: installer
+        run: echo ::set-output name=filename::streamlink-$(python setup.py --version | sed 's/+/_/')
+      - name: Build
+        run: ./script/makeinstaller.sh "${{ steps.installer.outputs.filename }}"
+      - name: Upload artifact
+        if: github.repository == 'streamlink/streamlink' && (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') || github.event_name == 'schedule')
+        uses: actions/upload-artifact@v2-preview
+        with:
+          name: ${{ steps.installer.outputs.filename }}-win32
+          path: ${{ env.STREAMLINK_DIST_DIR }}/${{ steps.installer.outputs.filename }}.exe
+
+  deploy-documentation:
+    name: Deploy docs
+    if: github.repository == 'streamlink/streamlink' && github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/'))
+    needs:
+      - test
+      - documentation
+      - windows-installer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 300
+      - name: Fetch tags
+        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          ./script/install-dependencies.sh
+          python -m pip install -r docs-requirements.txt
+      - name: Build
+        run: make --directory=docs html
+      - name: Deploy
+        env:
+          DOCS_KEY_PASSPHRASE: ${{ secrets.DOCS_KEY_PASSPHRASE }}
+        run: ./script/deploy-docs.sh
+
+  release:
+    name: New release
+    if: github.repository == 'streamlink/streamlink' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
+    needs:
+      - deploy-documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 300
+      - name: Fetch tags
+        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          ./script/install-dependencies.sh
+          python -m pip install --upgrade wheel twine
+      - name: Installer file name
+        id: installer
+        run: echo ::set-output name=filename::streamlink-$(python setup.py --version | sed 's/+/_/')
+      - name: Download installer artifact
+        uses: actions/download-artifact@v2-preview
+        with:
+          name: ${{ steps.installer.outputs.filename }}-win32
+          path: ${{ env.STREAMLINK_DIST_DIR }}
+      - name: sdist and wheels
+        env:
+          RELEASE_KEY_PASSPHRASE: ${{ secrets.RELEASE_KEY_PASSPHRASE }}
+        run: ./script/build-and-sign.sh
+      - name: Github release
+        env:
+          RELEASES_API_KEY: ${{ secrets.RELEASES_API_KEY }}
+        run: ./script/github_releases.py "${STREAMLINK_DIST_DIR}/"*".exe"
```

Comment:
Just noticed that the sdist file and signature don't get uploaded to the github release:
https://github.com/streamlink/streamlink/blame/master/.travis.yml#L96

Is this something we still want to do, or is it enough to publish this on PyPI?
```bash
curl -s 'https://api.github.com/repos/streamlink/streamlink/releases/latest' | jq '[.assets[] | {name:.name, count:.download_count}]'
```
```json
[
  {
    "name": "streamlink-1.3.1.exe",
    "count": 23947
  },
  {
    "name": "streamlink-1.3.1.tar.gz",
    "count": 1019
  },
  {
    "name": "streamlink-1.3.1.tar.gz.asc",
    "count": 158
  }
]
```

Reply:
Arch Linux as example uses a github URL
https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/streamlink#n15

so it might be useful

Reply:
Already fixed it, just rebuilding the test release before updating the PR to be extra sure.